jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/security/PropertyUserStore.java

```diff
@@ -127,21 +94,37 @@ public Resource getConfigResource()
      */
     public boolean isHotReload()
     {
-        return _hotReload;
+        return getReloadScanSeconds() > 0;
     }
 
     /**
      * Enable Hot Reload of the Property File
      *
-     * @param enable true to enable, false to disable
+     * @param enable true to enable to a 1 second scan, false to disable
+     * @see #setReloadScanSeconds(int)
      */
     public void setHotReload(boolean enable)
```

Comment:
I think this method should go away and be explicitly replaced with `setReloadScanSeconds(1)` calls instead.